frontend/src/metabase/admin/settings/components/widgets/EmbeddingCustomizationInfo.tsx

```diff
@@ -1,22 +1,16 @@
 import React from "react";
-import { jt } from "ttag";
+import { t, jt } from "ttag";
 import MetabaseSettings from "metabase/lib/settings";
 import ExternalLink from "metabase/core/components/ExternalLink";
 import SettingHeader from "../SettingHeader";
 
 export const EmbeddingCustomizationInfo = () => {
   const setting = {
-    display_name: "Customization",
-    description: (
-      <p style={{ maxWidth: "460px" }}>
-        {jt`Looking to remove the “Powered by Metabase” logo, customize colors
-        and make it your own? ${(
-          <ExternalLink href={MetabaseSettings.upgradeUrl()}>
-            Explore our paid plans.
-          </ExternalLink>
-        )}`}
-      </p>
-    ),
+    description: jt`In order to remove the Metabase logo from embeds, you can always upgrade to one of our ${(
+      <ExternalLink href={MetabaseSettings.upgradeUrl()}>
+        {t`paid plans.`}
```

Comment:
do you need to use `t` inside `jt` ?

Reply:
I've seen other places using this approach, to be honest, I'm not really sure 😅 . https://github.com/metabase/metabase/blob/92e721080f1dcac33145155adba089ee5f86c8a5/frontend/src/metabase/admin/settings/slack/components/SlackStatus/SlackStatus.tsx#L55-L57

@ranquild Can you confirm if this is ideal? I've seen other places break down the button component into its own thing and translate substring too. e.g.
- https://github.com/metabase/metabase/blob/e18353dfb1511cb5768ace850f65a9b9a423c3f0/frontend/src/metabase/admin/settings/components/widgets/EmbeddingLegalese.jsx#L13-L20
- https://github.com/metabase/metabase/blob/61bb4fc6a4378fe413434675ed51dadfbd851c56/frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.jsx#L78-L84

But from a quick search, most places seem to use `jt` to wrap the whole paragraph plus `t` inside an anchor tag just like what I do here.